app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -552,7 +552,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     public void onBackPressed() {
         if (isDrawerOpen()) {
-            drawerLayout.closeDrawer(navDrawer);
+            if (drawerLayout != null) {
```

Comment:
This is actually already covered by isDrawerOpen, but I guess SpotBugs doesn't detect this. Could you move the condition to the parent `if` statement please, to avoid the additional nested `if` statement?